webpack/assets/javascripts/react_app/routes/FiltersForm/FiltersForm.js

```diff
@@ -117,54 +117,44 @@ export const FiltersForm = ({ roleName, roleId, isNew, data, history }) => {
         defaultPermissions={data.permissions}
         setChosenPermissions={setChosenPermissions}
       />
-      {(showOrgs || showLocations) && (
-        <FormGroup
-          label={__('Override?')}
-          labelIcon={
-            <Popover
-              bodyContent={
-                <div>
-                  {__(
-                    'Filters inherit organizations and locations associated with the role by default. If override field is enabled, the filter can override the set of its organizations and locations. Later role changes will not affect such filter.After disabling the override field, the role organizations and locations apply again.'
-                  )}
-                </div>
-              }
-            >
-              <button
-                type="button"
-                aria-label="More info for override field"
-                onClick={e => e.preventDefault()}
-                className="pf-c-form__group-label-help"
-              >
-                <HelpIcon noVerticalAlign />
-              </button>
-            </Popover>
-          }
-        >
-          <Checkbox
-            ouiaId="override-checkbox"
-            isChecked={isOverride}
-            onChange={checked => {
-              setIsOverride(checked);
-            }}
-            aria-label="is override"
-            id="override-check"
-            name="override"
-          />
-        </FormGroup>
-      )}
-      {isOverride && (
-        <Taxonomies
-          showOrgs={showOrgs}
-          showLocations={showLocations}
-          setChosenOrgs={setChosenOrgs}
-          setChosenLocations={setChosenLocations}
-          defaultOrgs={data.organizations}
-          defaultLocations={data.locations}
+      <FormGroup
+        label={__('Organizations')}
+        helperText={__('Organizations are inherited from the role')}
+      >
+        <TextInput
+          ouiaId="filter-org-text-disabled"
+          aria-label="disabled organization field"
+          isDisabled
+          type="text"
+          value={chosenOrgs.join(' ,')}
         />
-      )}
+      </FormGroup>
+
+      <FormGroup
+        label={__('Locations')}
+        helperText={__('Locations are inherited from the role')}
+      >
+        <TextInput
+          aria-label="disabled location field"
+          ouiaId="filter-location-text-disabled"
+          isDisabled
+          type="text"
+          value={chosenLocations.join(' ,')}
+        />
+      </FormGroup>
       {isGranular ? (
         <>
+          {!!isTaxonomySearch && (
+            <>
+              <Alert
+                ouiaId="taxonomy-search-alert"
+                variant="info"
+                title={__(
+                  "The filter is scoped to the selected organizations and locations, therefore can't be unlimited"
```

Comment:
Not sure if helpful or just nitpicking, but I actually had trouble parsing what the alert meant. Do you think this would be better?

```suggestion
                  "You cannot use unlimited filtering because the filter is scoped to the organizations and locations inherited from the role."
```

I propose s/selected/inherited because 1. I think you don't really "select" orgs/locs anymore and 2. "inherited" is used in the descriptions on lines 122 and 135, I think consistency would help users connect the dots more quickly.

Reply:
Would it be better to move the info alert below the (greyed-out) unlimited checkbox?

Reply:
Yeah, for consistency should we change the warning to be a field small warning or an alert? 
The 2 options shown here:
![image](https://github.com/user-attachments/assets/e7ed1480-7240-4a5c-9463-9aa69738427e)

Reply:
@aneta-petrova ?

Reply:
Sorry, I have a hard time deciding which of the two options would be better. What do you think?

I guess what I feel most strongly about here is the wording -- as suggested above.